src/ProjectPortalPage/ProjectPortalPage.tsx

```diff
@@ -28,32 +28,40 @@ class ProjectPortalPage extends React.Component<
   };
 
   render() {
-    return (
-      <div className="project-portal__container">
-        <Header />
-        <div className="project-portal__carousel__background">
-          <div className="project-portal__carousel">
-            <Revisions mockups={this.props.currentProject.mockups} />
+    if (this.props.currentProject._id) {
+      return (
+        <div className="project-portal__container">
+          <Header />
```

Comment:
i might have missed this, but all references to header should be `<HeaderContainer />`